src/main/java/com/theelixrlabs/healthcare/constants/DoctorConstants.java

```diff
@@ -0,0 +1,14 @@
+package com.theelixrlabs.healthcare.constants;
+
+/**
+ * Constants related to doctors in the healthcare system.
+ */
+public class DoctorConstants {
+    public static final String DOCTORS_COLLECTION_NAME = "doctors";
+    public static final String FIRST_NAME_SHOULD_BE_MANDATORY = "First name is mandatory";
```

Comment:
Better to move the messages here to a message.properties file under the resources folder.

Reply:
moved the messages to a separate MessageConstants class.